samples/unit-tests/stock-tools/indicators/demo.js

```diff
@@ -42,89 +42,605 @@ QUnit.test('Managing tech indicators in Stock Tools', function (assert) {
+        0,
+        0.0001, // up to 0.0001% is fine
+        'Correct Top for Main axis after adding 1 indicators. Mixed axis heights'
+    );
+    assert.close(
+        parseFloat(chart.yAxis[1].options.height),
+        60 / 1.2,
+        0.0001, // up to 0.0001% is fine
+        'Correct Height for  second Main axis after adding 1 indicator. Mixed axis heights'
+    );
+
+    assert.close(
+        parseFloat(chart.yAxis[1].options.top),
+        20 / 1.2,
+        0.0001, // up to 0.0001% is fine
+        'Correct Top for second Main axis after adding 1 indicators. Mixed axis heights'
+    );
+    assert.close(
+        parseFloat(chart.yAxis[3].options.height),
+        20 / 1.2,
+        0.0001, // up to 0.0001% is fine
+        'Correct Height for  second Main axis after adding 1 indicator. Mixed axis heights'
+    );
+
+    assert.close(
+        parseFloat(chart.yAxis[3].options.top),
+        80 / 1.2,
+        0.0001, // up to 0.0001% is fine
+        'Correct Top for second Main axis after adding 1 indicators. Mixed axis heights'
+    );
+    assert.close(
+        parseFloat(chart.yAxis[3].options.height),
+        20 / 1.2,
+        0.0001, // up to 0.0001% is fine
+        'Correct Height for  second Main axis after adding 1 indicator. Mixed axis heights'
+    );
+
+    assert.close(
+        parseFloat(chart.yAxis[3].options.top),
+        80 / 1.2,
+        0.0001, // up to 0.0001% is fine
+        'Correct Top for second Main axis after adding 1 indicators. Mixed axis heights'
+    );
+
+    assert.close(
+        parseFloat(chart.yAxis[4].options.height),
+        20 / 1.2,
+        0.0001, // up to 0.0001% is fine
+        'Correct Height for  second Main axis after adding 1 indicator. Mixed axis heights'
+    );
+
+    assert.close(
+        parseFloat(chart.yAxis[4].options.top),
+        100 / 1.2,
+        0.0001, // up to 0.0001% is fine
+        'Correct Top for second Main axis after adding 1 indicators. Mixed axis heights'
+    );
+
+    chart.navigationBindings.utils.manageIndicators.call(
+        chart.navigationBindings,
+        {
+            actionType: 'remove',
+            seriesId:
+                chart.series[chart.series.length - 2]
+                    .options.id
+        }
+    );
+
+    assert.close(
+        parseFloat(chart.yAxis[0].options.height),
+        60,
+        0.0001, // up to 0.0001% is fine
+        'Correct Height for Main axis after adding 1 indicator. Mixed axis heights'
+    );
+
+    assert.close(
+        parseFloat(chart.yAxis[0].options.top),
+        0,
+        0.0001, // up to 0.0001% is fine
+        'Correct Top for Main axis after adding 1 indicators. Mixed axis heights'
+    );
+    assert.close(
+        parseFloat(chart.yAxis[1].options.height),
+        60,
+        0.0001, // up to 0.0001% is fine
+        'Correct Height for  second Main axis after adding 1 indicator. Mixed axis heights'
+    );
+
+    chart.destroy();
```

Comment:
3rd one

Reply:
I replaced the recreating of the chart with updates.